agent/signal_unix.go

```diff
@@ -0,0 +1,10 @@
+// +build !windows
+
+package agent
+
+import (
+	"os"
+	"syscall"
+)
+
+var forwardSignals []os.Signal = []os.Signal{os.Interrupt, syscall.SIGTERM}
```

Comment:
nitpick but I don't think you need the first `[]os.Signal` here. Surprised `fmt` doesn't remove this actually.